conformance/client/src/main/kotlin/com/connectrpc/conformance/client/Client.kt

```diff
@@ -228,17 +228,8 @@ class Client(
             }
         } catch (ex: Throwable) {
             if (!sent) {
-                val connEx = if (ex is ConnectException) {
-                    ex
-                } else {
-                    ConnectException(
-                        code = Code.UNKNOWN,
-                        message = ex.message,
-                        exception = ex,
-                    )
-                }
                 return ClientResponseResult(
-                    error = connEx,
+                    error = asConnectException(ex),
```

Comment:
I consolidated a few `if (ex is ConnectException) ...` blocks with a new helper function.